docs/specs/tripgo.swagger.yaml

```diff
@@ -540,6 +540,23 @@ paths:
           in: query
           description: Boolean if routing result should include only the best trip found for query.
           default: false
+        - name: cyclingAlt
+          type: string
```

Comment:
Would it make sense to turn this into an `array` instead, e.g., to provide `cyclingAlt=best&cyclingAlt=friendliest&shortest`. Would make it easier to add other alternatives down the track, and allow asking for any combinations of those.

Reply:
Yes... that crossed my mind too, I was undecided, I just thought the most common use would be `all`, or `best`, so I leaned towards the other option. But I agree that an `array` is more flexible and easier to add other alernatives.  I need to change the implementation a bit to support that, but it's not a big change. Should I go ahead, @nighthawk ?

Reply:
Also, could we still keep `all` as a shortcut?

Reply:
Sounds good. Don’t worry changing it to an array yet.